fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java

```diff
@@ -401,6 +390,25 @@ public void update(List<Pair<String, String>> properties, boolean isReplay) thro
         defaultCloudCluster = newDefaultCloudCluster;
     }
 
+    private String checkCloudDefaultCluster(String[] keyArr, String value, String defaultComputeGroup)
+            throws ComputeGroupException, DdlException {
+        // check cluster auth
+        if (!Env.getCurrentEnv().getAuth().checkCloudPriv(UserIdentity.fromString(qualifiedUser),
+                value, PrivPredicate.USAGE, ResourceTypeEnum.CLUSTER)) {
+            throw new ComputeGroupException(String.format("set default compute group failed, "
+                + "user {} must first have auth to use this compute group ", value),
```

Comment:
"user {} must first have auth to use this compute group "
-> "user {} has no permission to use compute group {}, please grant use privilege first with `grant USE_PRIV to ${user}`

Reply:
fixed